int-tests/vagrant/user-data

```diff
@@ -0,0 +1,37 @@
+#cloud-config
+
+---
+coreos:
+  etcd2:
+    discovery: https://discovery.etcd.io/e2e0adc537a5edb6e882ba0e2c65c234
```

Comment:
Let's not commit the discovery URL, you can leave this blank (e.g: `discovery: `)